api/main_endpoints/routes/Auth.js

```diff
@@ -81,15 +81,15 @@ router.post('/login', function(req, res) {
                 { email: user.email },
                 // update this field
                 { pagesPrinted: 0 },
-                function(error, result) {
-                  if (error) {
-                    return res.sendStatus(BAD_REQUEST);
-                  }
-
-                  if (result.nModified < 1) {
-                    return res.sendStatus(NOT_FOUND);
-                  }
-                }
+                // function(error, result) {
```

Comment:
is there a reason why this is commented out?

Reply:
This function was causing the server to crash every other time I compiled something in the api folder. Lawrence helped me with this on Slack a few days ago.

Reply:
got it, you can remove this code then